docs/Configuration.md

```diff
@@ -261,9 +261,28 @@ Jest will fail if:
 
 Default: `undefined`
 
-This option allows the use of a custom dependency extractor. It must be a node module that exports an object with an `extract` function expecting a string as the first argument for the code to analyze and Jest's dependency extractor as the second argument (in case you only want to extend it).
+This option allows the use of a custom dependency extractor. It must be a node module that exports an object with an `extract` function. E.g.:
 
-The function should return an iterable (`Array`, `Set`, etc.) with the dependencies found in the code.
+```javascript
+const fs = require('fs');
+const crypto = require('crypto');
+
+module.exports = {
+  extract(code, filePath, defaultExtract) {
```

Comment:
Should we pass an object instead of multiple args? Or `code` as first, then an object?

Reply:
I would've preferred that (I generally prefer objects than positional arguments as they're easier to extend) but I favored consistency with the rest of the Jest codebase.

Now that we're considering how the configuration should be from now on, we can reach an agreement about this.

This isn't urgent so we can discuss this in a wider scope before merging.

Reply:
My vote is an object :D